bin/container.sh

```diff
@@ -256,20 +271,23 @@ exec_commands_project() {
 }
 
 
-optspec=":f:p:v:s:h-:"
+optspec=":f:p:v:m::s:h-:"
```

Comment:
Nice catch. 👍  No specific reason. It is a typo but seemed to work fine functionally despite that. Removed now.